applications/crossbar/src/cb_apps_util.erl

```diff
@@ -63,11 +63,12 @@ is_authorized(AccountId, UserId, AppId) ->
             AppJObj = wh_json:get_value(AppId, kz_apps_store:apps(Doc)),
             Allowed = wh_json:get_value(<<"allowed_users">>, AppJObj, <<"specific">>),
             Users = wh_json:get_value(<<"users">>, AppJObj, []),
-            case {Allowed, Users} of
+            UserIds = [wh_json:get_value(<<"id">>, U, []) || U <- Users],
```

Comment:
Why default to `""`?

Also, shouldn't all occurrences of that default value be removed from the list? Matching on `[]` and `lists:member` look like they could be harmed by `undefined` being where it shouldn't be